test/dataSet.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np

```

Comment:
The data structures below are a bit ugly. I know, it's a leftover from the previous implementation.
Please think about a better way to do it in the future; not needed for this review to pass.